sysutils/xdg-utils/Portfile

```
@@ -16,16 +20,11 @@ long_description    Xdg-utils is a set of command line tools that \
                     assist applications with a variety of desktop integration tasks.
 
 homepage            https://www.freedesktop.org/wiki/Software/xdg-utils/
-master_sites        https://portland.freedesktop.org/download/
 
-checksums           rmd160  e09d258c26834555307d6edd6261514d546587ee \
-                    sha256  d798b08af8a8e2063ddde6c9fa3398ca81484f27dec642c5627ffcaa0d4051d9 \
-                    size    297170
+checksums           rmd160  81081208c96f2f023a58ddae6e0f281bf454020f \
+                    sha256  93d510dccf328378f012fe195b4574c2fac1cd65a74d0852d6eaa72e5a2065a7 \
+                    size    293386
```

With `gitlab.setup` now being used correctly, the `checksums` are different because the name of the enclosing directory in the automatically-generated tarball no longer contains the tag prefix:

```suggestion
checksums           rmd160  1e05be5815a756e5155fcd99222a42f5d2d6cfe6 \
                    sha256  3288aad73c764569e1d06d8368af28688bbba0549cf727e2278886fe66605c81 \
                    size    293209
```